README.md

```diff
@@ -79,6 +79,7 @@ In addition, many plugins can be found on the [npm registry](https://www.npmjs.c
   [omi-chart](https://github.com/Tencent/omi/tree/master/components/chart) | Omi | ✔ | ✔
   [react-chartjs-2](https://github.com/jerairrest/react-chartjs-2) | React | ✔ | ✔
   [vue-chartjs](https://github.com/apertureless/vue-chartjs/) | Vue.js | ✔ | ✔
+  [svelte-chartjs](https://github.com/SauravKanchan/svelte-chartjs) | Svelte | | ✔
```

Comment:
We have these in alphabetical order, so need to move up.

Reply:
@kurkle Sorry. Fixed.